siriuspy/siriuspy/devices/bpm.py

```diff
@@ -907,7 +920,9 @@ class FamBPMs(_DeviceSet):
 
     TIMEOUT = 10
     RFFEATT_MAX = 30
+    PROPERTIES_ACQ = BPM.PROPERTIES_ACQ
     PROPERTIES_DEFAULT = BPM.PROPERTIES_DEFAULT
+    ALL_MTURN_SIGNALS2ACQ = ('A', 'B', 'C', 'D', 'X', 'Y', 'Q', 'S')
```

Comment:
is there a reason to use capital letters?

Reply:
I think I'm used to think of BPM Antennas as A, B, C and D. Not a, b, c and d. Then I just kept the same standard for the positions.